src/components/Content/Animals/Animals.js

```diff
@@ -1,13 +1,154 @@
+const monkeyImg = 'https://i.pinimg.com/474x/27/72/77/277277945ff5b32cd3111d5c688deda7.jpg'
+const iguanaImg = 'https://upload.wikimedia.org/wikipedia/commons/thumb/6/62/Male_Green_Iguana_Belize.jpg/220px-Male_Green_Iguana_Belize.jpg'
 
 function Animals() {
-    return (
+
+    const[showCategories, setShowCategories] = useState(true);
+    const[showDogs, setShowDogs] = useState(false)
+    const[showCats, setShowCats] = useState(false)
+    const[showBirds, setShowBirds] = useState(false)
+    const[showMonkeys, setShowMonkeys] = useState(false)
+    const[showIguanas, setShowIguanas] = useState(false)
+
+    const cancel = () => {
+        setShowCategories(false);
+        setShowDogs(false);
+        setShowCats(false);
+        setShowBirds(false);
+        setShowMonkeys(false);
+        setShowIguanas(false);
+    }
+
+    const categoriesClickHandler = () => {
+        cancel();
+        setShowCategories(true);
+    }
+
+    const dogClickHandler = () => {
+        cancel();
+        setShowDogs(true);
+    }
+
+    const catClickHandler = () => {
+        cancel();
+        setShowCats(true);
+    }
+
+    const birdClickHandler = () => {
+        cancel();
+        setShowBirds(true);
+    }
+
+    const monkeyClickHandler = () => {
+        cancel();
+        setShowMonkeys(true);
+    }
+
+    const iguanaClickHandler = () => {
+        cancel();
+        setShowIguanas(true);
+    }
+
+    let animalContent = (
         <div className={styles.animals}>
-            All the dogs should have fancy hats 
+            <h1 className={styles.title}>Categories</h1>
+            <div onClick={dogClickHandler}>
+                <AnimalCategory animalimg={dogImg1} category={'Dogs'} />
+            </div>
+            <div onClick={catClickHandler}>
+                <AnimalCategory animalimg={catImg} category={'Cats'} />
+            </div>
+            <div onClick={birdClickHandler}>
+                <AnimalCategory animalimg={birdImg} category={'Birds'} />
+            </div>
+            <div onClick={monkeyClickHandler}>
+                <AnimalCategory animalimg={monkeyImg} category={'Monkeys'} />
+            </div>
+            <div onClick={iguanaClickHandler}>
+            <AnimalCategory animalimg={iguanaImg} category={'Iguanas'} />
+            </div>
             <br />
         </div>
     )
+
+    if(showDogs){
+        animalContent = (
+            <div>
+                <h1 className={styles.title}>Dogs</h1>
+                <AnimalElem animalimg={dogImg1} name={'George'} desc={'A cool dog!'} />
+                <AnimalElem animalimg={dogImg2} name={'Pam'} desc={'A stinky dog...'}/>
+                <AnimalElem animalimg={dogImg3} name={'Goofy'} desc={'A goofy dog.'}/>
+                <AnimalElem animalimg={dogImg4} name={'Max'} desc={'A singing dog.'}/>
+                <AnimalElem animalimg={dogImg5} name={'Daisy'} desc={'A pretty fast dog.'} /> 
```

Comment:
To avoid code duplication, you can render elements from an array. 
Ex

`const dogs = [{name: "Gerorge, desc: "A cool dog!, img: dogImg}, etc....]
/...

.../
return(
{dogs.map(dog => {
<AnimalElem animalimg={dog.img} name={dog.name} desc={dog.desc}/>
}
)}
)

`

This way you can more easly add more elements and also add new content as a admin!